e2e/libs/keywords/sharemanager_keywords.py

```
@@ -48,14 +48,32 @@ def wait_for_sharemanagers_deleted(self, name=[]):
+
+    def wait_for_share_manager_pod_running(self, name):
+        sharemanager_pod_name = "share-manager-" + name
+        retry_count, retry_interval = get_retry_count_and_interval()
+        for i in range(retry_count):
+            sharemanager_pod = get_pod(sharemanager_pod_name, "longhorn-system")
+            if sharemanager_pod.status.phase == "Running":
+                return
 
-    def wait_for_share_manager_running(self, name):
-        return self.sharemanager.wait_for_running(name)
+        assert False, f"sharemanager pod {sharemanager_pod_name} not running"
```

Ruff (0.7.0)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)